src/issues/issueFeatureRegistrar.ts

```diff
@@ -61,6 +66,29 @@ export class IssueFeatureRegistrar implements vscode.Disposable {
 		}
 	}
 
+	async createIssueFromFile() {
+		if (this.createIssueInfo === undefined) {
+			return;
+		}
+		let text: string;
+		if (!vscode.window.activeTextEditor) {
+			return;
+		}
+		text = vscode.window.activeTextEditor.document.getText();
+		const indexOfEmptyLine = text.indexOf('\n\n');
+		if (indexOfEmptyLine < 0) {
+			return;
+		}
+		const title = text.substring(0, indexOfEmptyLine);
+		const body = text.substring(indexOfEmptyLine + 2);

```

Comment:
should we trim off the explanation comment from the issue body?

Reply:
I would leave it, similar to how GitHub leaves the comment in the issue templates.